test/Libraries/PackageManagerTests/PackageManagerClientTests.cs

```diff
@@ -261,6 +261,21 @@ public void Upvote_ReturnsFalseWhenRequestThrowsException()
             Assert.IsFalse(res);
         }
 
+        //[Test]
```

Comment:
Uncomment this test? would be nice to test this. Is the mock test failing for any reason?